nestkernel/nest.h

```diff
@@ -82,6 +82,9 @@ ArrayDatum get_connections( const DictionaryDatum& dict );
 
 void simulate( const double& t );
 void resume_simulation();
+void run( const double& t );
+void prepare();
+void cleanup();
```

Comment:
Since this is the high-level API, the functions should be document with doxygen comments.
And to we still need `resume_simulation()` after this split?